hubmap/frontend/src/components/home/LogInStatus.js

```diff
@@ -153,21 +153,25 @@ export default function LoggedInStatus() {
   if (email === '' || email === undefined) {
     return (
       <div className={classes.sectionMenuDesktop}>
-        <a href={`${API_URL}auth/login/globus/`} style={{ textDecoration: 'none' }}>
-
-          <Button color={grey[300]} aria-haspopup="true">
+        <a id="globuslogin" href={`${API_URL}auth/login/globus/`} style={{ textDecoration: 'none' }}>
+          <Button color={grey[300]} aria-haspopup="true" id="button-login">
```

Comment:
- Since we shouldn't depend on IDs like this in cypress tests, please remove.
- In the future, if we do need an element ID, please use dashes: `globus-login` not `globuslogin`.

Reply:
This comment is referring to specific commit. but most recent commit, does not have this ids.